OracleRestDataServices/dockerfiles/3.0.10/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# LICENSE CDDL 1.0 + GPL 2.0
+#
+# Copyright (c) 1982-2017 Oracle and/or its affiliates. All rights reserved.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Rest Data Services
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) ords.3.0.10.165.06.53.zip
+#     Download Oracle Rest Data Services from
+#     http://www.oracle.com/technetwork/developer-tools/rest-data-services/downloads/index.html
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put the downloaded file in the same directory as this Dockerfile
+# Run: 
+#      $ docker build -t oracle/restdataservices:3.0.10 . 
+#
+# Pull base image
+# ---------------
+FROM openjdk:8
```

Comment:
We don't support the official OpenJDK image. Is it possible to use the Server JRE 8 image instead?